packages/app/src/Charcoal/App/Event/AbstractEventListener.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Charcoal\App\Event;
+
+use Pimple\Container;
+use Psr\Log\LoggerAwareTrait;
+
+/**
+ * Abstract Event Listener
+ */
+abstract class AbstractEventListener implements EventListenerInterface
+{
+    use LoggerAwareTrait;
+
+    /**
+     * @param Container $container Pimple DI Container.
+     * @return void
+     */
+    public function setDependencies(Container $container)
+    {
+        $this->setLogger($container['logger']);
+    }
```

Comment:
Is logger awareness necessary for the base listener?

Reply:
I think every listener should log something when they are being called.